schedule/security/389ds_sssd_client.yaml

```diff
@@ -0,0 +1,8 @@
+name:    389ds_sssd_client
+description:    >
+    This is for 389ds and sssd authentication
```

Comment:
I see that you use the **389dc** in the code, so I think the description here needs to be changed, right?

Reply:
Thanks, let me fix it